web3.js/src/connection.ts

```diff
@@ -4278,6 +4278,27 @@ export class Connection {
     }
   }
 
+  /**
+   * get the stake minimum delegation
+   */
+  async getStakeMinimumDelegation(): Promise<RpcResponseAndContext<number>> {
```

Comment:
Let's let people specify a commitment that overrides the connection's default commitment.

1. Take an optional config here, that contains an optional `commitment` property.
2. Use `_buildArgs` with that `Commitment | undefined`. Internally, `_buildArgs` falls back to `this.commitment` if there's no override.